### DISCUSSION TITLE
hyprland.conf issues after last system and hyprdots updates

### DISCUSSION THREAD

**Discussion by psilvaricardo** [prasanthrangan/hyprdots#958] [Q&A] [answered]
Hi All, good night.

After updating my system and reinstalling the latest changes from this project, I have started to see some issues. Is someone else experiencing this? 

![image](https://github.com/prasanthrangan/hyprdots/assets/125488497/eb5a8c45-a762-490a-9f4f-fe4d4da2fb57)

**Comment by kRHYME7** [comment, marked as answer]
You're using the ``` hyprland-git ``` ?  If so, check upstream hyprland wiki as hyprdots follows the tagged release of ```  hyprland ``` version. 

If you want to revert ``` yay -Sy hyprland ```   

This are the changes to be merged soon if next tagged release is available. 
https://github.com/prasanthrangan/hyprdots/pull/930

**Reply by psilvaricardo** [reply]
@kRHYME7 Yes, I am using hyprland-git; thanks for your help.
Are you suggesting to install hyprland and ditch hyprland-git ?

I have just update it:

Package (1)     Old Version                    New Version                    Net Change
hyprland-git   0.35.0.r94.f534ac3f-1  0.35.0.r96.54a83299-1    0.00 MiB
